assets/javascript/health-check.js

```diff
@@ -112,4 +112,70 @@ jQuery(document).ready(function ($) {
 			$button.text( health_check.string.copied );
 		}
 	});
-});
\ No newline at end of file
+
+	$( '#health-check-file-integrity' ).submit( function( e ) {
+		e.preventDefault();
+		$( '#tools-response-holder' ).html( '<span class="spinner"></span>' );
+		$( '#tools-response-holder .spinner' ).addClass( 'is-active' );
+
+		var data = {
+			'action': 'health-check-files-integrity-check'
+		};
+
+		$.post(
+			ajaxurl,
+			data,
+			function( response ) {
+				$( '#tools-response-holder .spinner' ).removeClass( 'is-active' );
+				$( '#tools-response-holder').html( response.data.message );
+		});
+	});
+
+	$( '#health-check-mail-check' ).submit( function( e ) {
+		e.preventDefault();
+		var email = $('#health-check-mail-check #email').val();
+		$( '#tools-response-holder' ).html( '<span class="spinner"></span>' );
+		$( '#tools-response-holder .spinner' ).addClass( 'is-active' );
+
+		var data = {
+			'action': 'health-check-mail-check',
+			'email' : email
+		};
+
+		$.post(
+			ajaxurl,
+			data,
+			function( response ) {
+				console.log(response.data.message);
```

Comment:
We should remove debug lines (IE will halt execution of JS if debug lines are left in without the console window open)